docs/components/clients/users.ts

```diff
@@ -524,6 +524,6 @@ export const users: Array<TurboUser> = [
     pinned: true,
     style: {
       width: 150,
-    }
+    },
```

Comment:
Unrelated change?

Reply:
yeah although this is a prettier fix which should be triggered by the pre-commit hook. Probably should merge it in because it'll show up on any other PR